src/backend/base/langflow/components/jigsawstack/ai_scrape.py

```
@@ -0,0 +1,114 @@
+        try:
+            client = JigsawStack(api_key=self.api_key)
+            
+            #build request object
+            scrape_params = {}
+            if self.url:
+                scrape_params["url"] = self.url
+            if self.html:
+                scrape_params["html"] = self.html
+
+            if len(scrape_params["url"].strip(" ")) == 0 and len(scrape_params["html"].strip()) == 0:
+                raise ValueError("Either 'url' or 'html' must be provided for scraping")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)